### PR TITLE
home page of note app

### PR DESCRIPTION
**Pull request by devAmlan**
Made the home page of my note app.
Integrated Firebase as database.
Built some features like creating notes, Pin, archive, trash, color, label the notes.
demo ->
squinote-gif-1

### CODE REVIEW

File: frontend/src/component/notearea/NoteArea.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import Notes from '../notes/Notes'
+
+function NoteArea() {
+  return (
+    <>
+    <div className="notes_area">
+    <Searchbar/>
+    <Noteinput/>
+    <Notes/>
+    </div>
```
Comment (iRohitGaur) [inline frontend/src/component/notearea/NoteArea.js:14]: suggestion: use prettier for auto indentation

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by srishtigautam21 — APPROVED** [review]
The UI looks great
All the functionalities seem to work smoothly
Good work...good to merge

File: frontend/src/component/notearea/NoteArea.js (hunk 1 above)
Comment (devAmlan) [reply frontend/src/component/notearea/NoteArea.js:14]: Thanks for the feedback Rohit, I will use prettier.

**Pull request merged**